test/test_fake_tensor.py

```diff
@@ -159,11 +159,31 @@ def test_non_parameter_grad(self):
         TEST_WITH_TORCHDYNAMO, "isinstance check for FakeTensor won't work with compile"
     )
     @unittest.skipIf(not RUN_CUDA, "requires cuda")
-    def test_index_cuda_with_cpu(self):
+    @parametrize(
+        "dtype",
```

Comment:
can we use something from this file https://github.com/pytorch/pytorch/blob/bdd942efd76e74baa5dd0a262f7c843ddfe2e11b/torch/testing/_internal/common_dtype.py ?

Reply:
ah this is useful, thanks - done.